ocp_resources/resource.py

```diff
@@ -107,13 +108,26 @@ def get_client(
             )
         )
     try:
-        # Ref: https://github.com/kubernetes-client/python/blob/v26.1.0/kubernetes/base/config/__init__.py
-        LOGGER.info("Trying to get client via new_client_from_config")
-
         # kubernetes.config.kube_config.load_kube_config sets KUBE_CONFIG_DEFAULT_LOCATION during module import.
         # If `KUBECONFIG` environment variable is set via code, the `KUBE_CONFIG_DEFAULT_LOCATION` will be None since
         # is populated during import which comes before setting the variable in code.
         config_file = config_file or os.environ.get("KUBECONFIG", "~/.kube/config")
+        proxy = os.environ.get("HTTPS_PROXY") or os.environ.get("HTTP_PROXY")
+
+        if proxy:
```

Comment:
`new_client_from_config` get `client_configuration` so proxy can be set there

Reply:
proxy can't be set after the client already created, because the client need the proxy to be set before creating the client in the configuration, because it's using the proxy configuration for low level libraries(urllib3) that require it for initialization when the  creating the client.

Reply:
Seems to me that it should work

```
In [19]: proxy = "https://localost.com"

In [20]: client_configuration = client.Configuration()

In [21]: client_configuration.proxy = proxy

In [22]: qq = kubernetes.dynamic.DynamicClient(client=kubernetes.config.new_client_from_config(config_file=config_file, client_configuration=client_configuration))

In [23]: qq.configuration.proxy
Out[23]: 'https://localost.com'
```

Reply:
Ohh got it now, done, thanks!